[CI] SmokeTestMultiNodeClientYamlTestSuiteIT class failing

**Build Scans:**
- [elasticsearch-pull-request #29099 / part-1](https://gradle-enterprise.elastic.co/s/ry4hj4uzkqifq)
- [elasticsearch-intake #9104 / part1](https://gradle-enterprise.elastic.co/s/yhfsyu5jffmd4)
- [elasticsearch-pull-request #28640 / part-1](https://gradle-enterprise.elastic.co/s/hbiendvrqj6j4)

**Reproduction Line:**
```
./gradlew ":qa:smoke-test-multinode:yamlRestTest" --tests "org.elasticsearch.smoketest.SmokeTestMultiNodeClientYamlTestSuiteIT.test {yaml=index/91_metrics_no_subobjects/Root without subobjects with synthetic source}" -Dtests.seed=1ECE5C4F37762C4 -Dtests.locale=de-LU -Dtests.timezone=Pacific/Chuuk -Druntime.java=22
```

**Applicable branches:**
main

**Reproduces locally?:**
N/A

**Failure History:**
[See dashboard](https://es-delivery-stats.elastic.dev/app/dashboards#/view/dcec9e60-72ac-11ee-8f39-55975ded9e63?_g=(refreshInterval:(pause:!t,value:60000),time:(from:now-7d%2Fd,to:now))&_a=(controlGroupInput:(chainingSystem:HIERARCHICAL,controlStyle:twoLine,ignoreParentSettings:(ignoreFilters:!f,ignoreQuery:!f,ignoreTimerange:!f,ignoreValidations:!t),panels:('0c0c9cb8-ccd2-45c6-9b13-96bac4abc542':(explicitInput:(dataViewId:fbbdc689-be23-4b3d-8057-aa402e9ed0c5,enhancements:(),fieldName:task.keyword,grow:!t,id:'0c0c9cb8-ccd2-45c6-9b13-96bac4abc542',searchTechnique:wildcard,selectedOptions:!(),singleSelect:!t,title:'Gradle%20Task',width:medium),grow:!t,order:0,type:optionsListControl,width:small),'144933da-5c1b-4257-a969-7f43455a7901':(explicitInput:(dataViewId:fbbdc689-be23-4b3d-8057-aa402e9ed0c5,enhancements:(),fieldName:name.keyword,grow:!t,id:'144933da-5c1b-4257-a969-7f43455a7901',searchTechnique:wildcard,selectedOptions:!('org.elasticsearch.smoketest.SmokeTestMultiNodeClientYamlTestSuiteIT'),title:Test,width:medium),grow:!t,order:2,type:optionsListControl,width:medium),'4e6ad9d6-6fdc-4fcc-bf1a-aa6ca79e0850':(explicitInput:(dataViewId:fbbdc689-be23-4b3d-8057-aa402e9ed0c5,enhancements:(),fieldName:className.keyword,grow:!t,id:'4e6ad9d6-6fdc-4fcc-bf1a-aa6ca79e0850',searchTechnique:wildcard,selectedOptions:!('org.elasticsearch.smoketest.SmokeTestMultiNodeClientYamlTestSuiteIT'),title:Suite,width:medium),grow:!t,order:1,type:optionsListControl,width:medium)))))

**Failure Message:**
```
java.lang.AssertionError: Failure at [index/91_metrics_no_subobjects:201]: got unexpected warning header [
	299 Elasticsearch-8.16.0-915528c00e9f5de9f74c86c2e775b96c60eced7f "index template [test] has index patterns [test-*] matching patterns from existing older templates [global] with patterns (global => [*]); this template [test] will take precedence during new index creation"
]

```

**Issue Reasons:**
- [main] 2 consecutive failures in class org.elasticsearch.smoketest.SmokeTestMultiNodeClientYamlTestSuiteIT
- [main] 3 failures in class org.elasticsearch.smoketest.SmokeTestMultiNodeClientYamlTestSuiteIT (0.5% fail rate in 580 executions)
- [main] 2 failures in step part-1 (1.4% fail rate in 148 executions)
- [main] 2 failures in pipeline elasticsearch-pull-request (1.3% fail rate in 150 executions)

**Note:**
This issue was created using new test triage automation. Please report issues or feedback to es-delivery.

--- a/muted-tests.yml
+++ b/muted-tests.yml
@@ -83,8 +83,6 @@ tests:
 - class: org.elasticsearch.xpack.ml.integration.MlJobIT
   method: testDeleteJobAfterMissingIndex
   issue: https://github.com/elastic/elasticsearch/issues/112088
-- class: org.elasticsearch.smoketest.SmokeTestMultiNodeClientYamlTestSuiteIT
-  issue: https://github.com/elastic/elasticsearch/issues/112147
 - class: org.elasticsearch.smoketest.WatcherYamlRestIT
   method: test {p0=watcher/usage/10_basic/Test watcher usage stats output}
   issue: https://github.com/elastic/elasticsearch/issues/112189

